lib/extension.js

```diff
@@ -1,11 +1,11 @@
 const path = require('path');
 
-const EXTRE = /^[.]?[^.]+([.].*)$/;
+const EXTRE = /(\.[^.]+)+$/;
```

Comment:
Cc @cowboy to the rescue?
On Jun 4, 2015 7:59 PM, "Blaine Bublitz" <notifications@github.com> wrote:

> In lib/extension.js
> <https://github.com/tkellen/js-rechoir/pull/26#discussion_r31778089>:
>
> > @@ -1,11 +1,11 @@
> >  const path = require('path');
> >
> > -const EXTRE = /^[.]?[^.]+([.].*)$/;
> > +const EXTRE = /(\.[^.]+)+$/;
>
> I'm really bad with regexps, sorry.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tkellen/js-rechoir/pull/26/files#r31778089>.
>